alias/config.c

```diff
@@ -70,5 +70,5 @@ struct ConfigDef AliasVars[] = {
  */
 bool config_init_alias(struct ConfigSet *cs)
 {
-  return cs_register_variables(cs, AliasVars, DT_NO_VARIABLE);
+  return cs_register_variables(cs, AliasVars, 0);
```

Comment:
I see the pattern of `cs_register_variables(cs, ..., 0);` throughout all the `config.c` files. However, I don't see a single usage of `cs_register_variables()` that doesn't provide `0`. Do we need to keep it?

Reply:
The param was added for the the `NO_VAR` case, but I had other ideas for it.
My favourite was `DT_DISABLED` -- mostly a developer feature, but might be useful for packagers.

`DT_DISABLED` config would be accepted when reading a config file, but have no effect.
Libraries that were disabled by `./configure` would register their config, but no other code would be built.

As a developer this would allow you to have large config files that are quietly accepted when you don't have features enabled.

(disabled config would be logged)